packages/app-page-builder/src/admin/views/BlockCategories/IconPicker.tsx

```diff
@@ -110,9 +110,8 @@ const IconPicker = ({ value, onChange, label, description, validation }: IconPic
     }, [mustRenderGrid]);
 
     const onFilterChange = useCallback(
-        (value, cb) => {
```

Comment:
Why did you remove the `cb`?

Reply:
Previously we had [empty function call](https://github.com/webiny/webiny-js/pull/3788/files/3f81da807c561eb0a215e689a1a5b842b2b5c083#diff-22f0fc472ecfa63e00eb2362b7e34e5b41b1eca5df254d624d3b9fd983188123L115), but it requires 2 arguments as defined [here](https://github.com/webiny/webiny-js/blob/next/packages/ui/src/DelayedOnChange/DelayedOnChange.ts#L7).

Reply:
I still dont understand why did you remove it. When I go through the code, i found that cb is sometimes sent, so it should be executed.